TrafficJam/Forms/EntryPointsSub.cs

```diff
@@ -0,0 +1,50 @@
+﻿#region
+
+using System.Drawing;
+using System.Windows.Forms;
+
+#endregion
+
+namespace TrafficJam.Forms
+{
+    internal class EntryPointsSub : Panel
+    {
+        #region Fields
+
+        private readonly Font _trebuchetSmall = new Font( "Trebuchet MS", 8f );
+        private readonly Font _trebuchetLarge = new Font( "Trebuchet MS", 11f );
+
+        #endregion
+
+        public EntryPointsSub( int y )
```

Comment:
```suggestion
        public EntryPointsSub( EntryPointsPanel entryPointsPanel, int y )
```

This subpanel also needs to be aware of his owner, so they can report back to the application when a value changes.